internal/db/schema/migrations/oss/postgres/78/01_deleted_tables_and_triggers.up.sql

```diff
@@ -104,6 +278,11 @@ begin;
     return old;
   end;
   $$ language plpgsql;
+  comment on function insert_deleted_id is
+    'insert_deleted_id is a function that inserts a record into the table '
+    'specified by the first trigger argument. It takes the public ID from the row '
+    'affected by the trigger and the current timestamp. It is used to populate rows '
+    'of the deleted tables.';
 
   create function get_deletion_tables() returns setof name
```

Comment:
Is this function needed outside of this migration? If not can we drop it at the end of the migration?

Reply:
It gets used [here](https://github.com/hashicorp/boundary/blob/bb0536a0cbdc0c5f8afb2e03cf53f18f76d0e2c2/internal/pagination/purge/purge.go#L29) as well, where it goes through all the deletion tables and deletes records older than 30 days